internal/transform/ledger_test.go

```diff
@@ -78,24 +87,24 @@ func makeLedgerTestOutput() (output LedgerOutput, err error) {
 		OperationCount:             10,
 		SuccessfulTransactionCount: 1,
 		FailedTransactionCount:     1,
-		TxSetOperationCount:        "13",
+		TxSetOperationCount:        "2",
```

Comment:
How was the original test 13? The hardCodedTxSet was always 2?

Reply:
hmmmm it should actually be 13. looking into this